docs/utilities/parameters.md

```diff
@@ -4,29 +4,40 @@ description: Utility
 ---
 
 
-The parameters utility provides a way to retrieve parameter values from
+The parameters utilities provide a way to retrieve parameter values from
 [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html), 
 [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/), or [Amazon DynamoDB](https://aws.amazon.com/dynamodb/). 
-It also provides a base class to create your parameter provider implementation.
-
+ 
 **Key features**
 
-* Retrieve one or multiple parameters from the underlying provider
+* Retrieve one or multiple parameters from an underlying provider in a standard way
 * Cache parameter values for a given amount of time (defaults to 5 seconds)
 * Transform parameter values from JSON or base 64 encoded strings
 
 ## Install
-Depending on your version of Java (either Java 1.8 or 11+), the configuration slightly changes.
+In order to provide lightweight dependencies, each parameters module is available as its own
+package:
+
+* **Secrets Manager** - `powertools-parameters-secrets` - [A service that centrally manages the lifecycle of secrets](https://aws.amazon.com/secrets-manager/). Start here if you need to manage secrets only. 
+* **SSM Parameter Store** - `powertools-parameters-ssm` - [A secure, hierarchical store for configuration data management](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html). Start here if you need to manage various types of configuration data. 
+* **Amazon DynamoDB** -`powertools-parameters-dynamodb` - [A fast, flexible, NoSQL database](https://aws.amazon.com/dynamodb/)
+* **AWS AppConfig** - `powertools-parameters-appconfig` - [An application focussed configuration store](https://aws.amazon.com/systems-manager/features/appconfig) - start here if you are interested in advanced use cases, such as incremental deployment of configuration changes
+
+You can easily mix and match parameter providers within the same project for different needs.  
+
+Depending on which Java version you are using, you configuration will differ. Note that you must also provide
```

Comment:
this TODO thing is also not super clear, see my proposition below

Reply:
Fixed!